src/ert/scheduler/lsf_driver.py

```diff
@@ -102,6 +103,35 @@ def parse_bjobs(bjobs_output: str) -> Dict[str, Dict[str, Dict[str, str]]]:
     return {"jobs": data}
 
 
+def parse_resource_requirement_string(
```

Comment:
I feel something is off with the naming of this function, as its input is more than a single string.

Reply:
Since it is returning a single string, it is not a parse function, although it is doing parsing inside.